## 扩增子相关 | amplicon

### 工具 | tools
- [[UPARSE](http://drive5.com/uparse/)] - [C++/C, v12.0-beta1, 2024.6] - [2013.8, _Nat Methods_] - [[UPARSE: Highly accurate OTU sequences from microbial amplicon reads.](https://doi.org/10.1038/nmeth.2604)]
- [[Mothur](https://github.com/mothur/mothur/tree/v1.48.1)] - [C++, v1.48.1, 2024.5] - [2009.12, _Appl Environ Microbiol_] - [[Introducing mothur: Open-source, platform-independent, community-supported software for describing and comparing microbial communities.](https://doi.org/10.1128/AEM.01541-09)]
- [[dada2](https://github.com/benjjneb/dada2)] - [R, 1.26, 2022.11] - [2016.05, _Nat Methods_] [[DADA2: High resolution sample inference from Illumina amplicon data.](https://doi.org/10.1038/nmeth.3869)]
- [[QIIME2](https://qiime2.org/)] - [JAVA, 2024-05, 2024.05] - [2019.07, _Nat Biotechnol_] - [[Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2.](https://doi.org/10.1038/s41587-019-0209-9)]
- [[RDP Classifier](https://sourceforge.net/projects/rdp-classifier/)] - [Java, 2.14, 2023.08] - [2024.03, _Microb Ecol_] - [[Updated RDP taxonomy and RDP Classifier for more accurate taxonomic classification.](https://doi.org/10.1128/mra.01063-23)]
- [[Tax4Fun](https://tax4fun.gobics.de)] - [R, v0.3.1, 不再更新了] - [2015.05, _Bioinformatics_] - [Tax4Fun: predicting functional profiles from metagenomic 16S rRNA data](http://dx.doi.org/10.1093/bioinformatics/btv287)
- [Tax4Fun2] - [链接无效了] - [2020.05, _Environ Microbiome_] - [[Tax4Fun2: prediction of habitat-specific functional profiles and functional redundancy based on 16S rRNA gene sequences](https://doi.org/10.1186/s40793-020-00358-7)]
- [[PICRUSt](https://github.com/picrust/picrust)] - [Python, v1.1.4, 2019.06] - [2013.08, _Nat Biotechnol_] - [[Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences.](https://doi.org/10.1038/nbt.2676)]
- [[PICRUSt2](https://github.com/picrust/picrust2)] - [Python, v2.5.3, 2024.08] - [2020.06, _Nat Biotechnol_] - [[PICRUSt2 for prediction of metagenome functions.](https://doi.org/10.1038/s41587-020-0548-6)]
- [[graftM](https://github.com/geronimp/graftM)] - [Python, v0.14.0, 2022.5] - [2018.6, _Nucleic Acids Res_] - [[GraftM: a tool for scalable, phylogenetically informed classification of genes within metagenomes.](https://doi.org/10.1093/nar/gky174)]


### 数据库 | databases
- [[EUKARYOME](https://eukaryome.org)] - [18S/ITS/28S, v1.9.2, 2024.08] - [2024.06, _Database_] - [[EUKARYOME: the rRNA gene reference database for identification of all eukaryotes.](https://doi.org/10.1093/database/baae043)]
- [[Silva](https://www.arb-silva.de)] - [16S, v138.2, 2024.07] - [2013.1, _Nucleic Acids Res_] - [[The SILVA ribosomal RNA gene database project: improved data processing and web-based tools.](https://doi.org/10.1093/nar/gks1219)]
- [[Greengenes2](http://ftp.microbio.me/greengenes_release/)] - [16S, v2022.10, 2022.10] - [2023.07, _Nat Biotechnol_] - [[Greengenes2 unifies microbial data in a single reference tree.](https://doi.org/10.1038/s41587-023-01845-1)]
- [[MiDAS](https://midasfieldguide.org/guide)] - [16S, v5, 2024.07] - [2024.06, _Nat Commun_] - [[MiDAS 5: Global diversity of bacteria and archaea in anaerobic digesters.](https://doi.org/10.1038/s41467-024-49641-y)]
