
## implemented workflows and platforms for **metagenomes**
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [Aviary](https://github.com/rhysnewell/aviary) | Python | v0.9.2 | 2024.09 ||| a snakemake pipeline for assembly | binning | annotation | and strain diversity analysis
| [MAG](https://github.com/nf-core/mag) | Nextflow | v3.0.3 | 2024.8 | _NAR Genom Bioinform_ | 2022.2 | [nf-core/mag: a best-practice pipeline for metagenome hybrid assembly and binning](https://doi.org/10.1093/nargab/lqac007)
| [MetaWRAP](https://github.com/bxlab/metaWRAP) | Shell/Python | v1.3 | 2020.08 | _Microbime_ | 2018.09 | [MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis.](https://doi.org/10.1186/s40168-018-0541-1) |
| [VEBA](https://github.com/jolespin/veba) | Python | v2.3.0 | 2024.9 | _BMC Bioinform_ | 2022.10 | [VEBA: a modular end-to-end suite for in silico recovery, clustering, and analysis of prokaryotic, microeukaryotic, and viral genomes from metagenomes](https://doi.org/10.1186/s12859-022-04973-8) |
| [ATLAS](https://github.com/metagenome-atlas/atlas) | Python | v2.19.0 | 2024.7 | _BMC Bioinform_ | 2020.6 | [ATLAS: a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data](https://doi.org/10.1186/s12859-020-03585-4)
| [anvi'o](https://github.com/merenlab/anvio/tree/v8) | Win/MacOS/Linux | v8 | 2023.09 | _Nat Microbiol_ | 2020.12 | [Community-led, integrated, reproducible multi-omics with anvi’o.](https://doi.org/10.1038/s41564-020-00834-3)
| [nano-rave](https://github.com/sanger-pathogens/nano-rave) | Nextflow | v1.0.0 | 2023.01 | _Nat Microbiol_ | 2023.11 | [Drug resistance and vaccine target surveillance of Plasmodium falciparum using nanopore sequencing in Ghana](https://doi.org/10.1038/s41564-023-01516-6)
| [metagWGS](https://forgemia.inra.fr/genotoul-bioinfo/metagwgs) | Nextflow | v2.4.2 | 2023.6 | bioRxiv | 2024.9 | [metagWGS, a comprehensive workflow to analyze metagenomic data using Illumina or PacBio HiFi reads](https://doi.org/10.1101/2024.09.13.612854)
| [SqueezeMeta](https://github.com/jtamames/SqueezeMeta) | C/C++/Python/Perl | v1.6.5 | 2024.8 | _Front Microbiol_ | 2019.1 | [SqueezeMeta, A Highly Portable, Fully Automatic Metagenomic Analysis Pipeline](https://doi.org/10.3389/fmicb.2018.03349)
| [slamM](https://github.com/Ecogenomics/slamM/tree/master) | Snakemake
| [CAT_pack](https://github.com/MGXlab/CAT_pack) | Python | v6.0.1 | 2024.3 | _Genome Biol_ | 2019.10 | [Robust taxonomic classification of uncharted microbial sequences and bins with CAT and BAT.](https://doi.org/10.1186/s13059-019-1817-x) |
| [Metagenomics-Toolkit](https://github.com/metagenomics/metagenomics-tk) | Nextflow | v0.4.5 | 2024.10 | bioRxiv | 2024.10 | [Metagenomics-Toolkit: The Flexible and Efficient Cloud-Based Metagenomics Workflow featuring Machine Learning-Enabled Resource Allocation](https://doi.org/10.1101/2024.10.22.619569)
|| no code link seen ||| bioRxiv | 2024.9 | [An efficient pipeline for creating metagenomic-assembled genomes from ancient oral microbiomes](https://doi.org/10.46471/gigabyte.132)
| [BugBuster](https://github.com/gene2dis/BugBuster) | Nextflow ||| bioRxiv | 2025.2 | [BugBuster: A novel automatic and reproducible workflow for metagenomic data analysis](https://doi.org/10.1101/2025.02.24.639915)
| [MARTi](https://marti.cyverseuk.org/) |||||| [MARTi: a real-time analysis and visualisation tool for nanopore metagenomics](https://doi.org/10.1101/2025.02.14.638261)


## implemented workflows and platforms for **isolates**
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [Bactopia](https://github.com/bactopia/bactopia) | Nextflow/Perl | v3.1.0 | 2024.9 | _mSystems_ | 2020.8 | [Bactopia: a flexible pipeline for complete analysis of bacterial genomes.](https://doi.org/10.1128/mSystems.00190-20) |
| [ASA³P](https://github.com/oschwengers/asap) | Groovy/JS | v1.3.0 | 2020.5 | _PLoS Comput Biol_ | 2020.5 | [ASA³P: An automatic and scalable pipeline for the assembly, annotation and higher-level analysis of closely related bacterial isolates.](https://doi.org/10.1371/journal.pcbi.1007134) |
| [microPIPE](https://github.com/BeatsonLab-MicrobialGenomics/micropipe) | html tutorial Nextflow | step by step || _BMC Genomics_ | 2021.6 |[MicroPIPE: validating an end-to-end workflow for high-quality complete bacterial genome construction.](https://doi.org/10.1186/s12864-021-07767-z) |
| [AQUAMIS](https://gitlab.com/bfr_bioinformatics/AQUAMIS) | Python/Shell | v1.4.2 | 2024.6 | _Genes_ | 2021.4 | [Species-Specific Quality Control, Assembly and Contamination Detection in Microbial Isolate Sequences with AQUAMIS.](https://doi.org/10.3390/genes12050644)
| [Nullarbor](https://github.com/tseemann/nullarbor) | Perl | v1.41 | 2018.6 ||| Pipeline to generate complete public health microbiology reports from sequenced isolates. |
| [ProkEvo](https://github.com/npavlovikj/ProkEvo) | jupyter tutorial ||| _PeerJ_ | 2021.5 | [ProkEvo: an automated, reproducible, and scalable framework for high-throughput bacterial population genomics analyses.](https://doi.org/10.7717/peerj.11376)
| [Public Health Bioinformatics](https://github.com/theiagen/public_health_bioinformatics) | WDL | v2.2.1 | 2024.9 | _Microb Genom_ | 2023.7 | [Accelerating Bioinformatics Implementation in Public Health](https://doi.org/10.1099/mgen.0.001051) | TheiaCoV(viral), TheiaProk(bacterial) |
| [Public Health Bioinformatics](https://github.com/theiagen/public_health_bioinformatics) | WDL | v2.2.1 | 2024.9 | _Front Public Health_ | 2023.8 | [TheiaEuk: A Species-Agnostic Bioinformatics Workflow for Fungal Genomic Characterization.](https://doi.org/10.3389/fpubh.2023.1198213) |
| [rMAP](https://github.com/GunzIvan28/rMAP) | Shell/Python | tutorial || _Microb Genom_ | 2021.6 | [rMAP: the Rapid Microbial Analysis Pipeline for ESKAPE bacterial group whole-genome sequence data.](https://doi.org/10.1099/mgen.0.000583)
| [TORMES](https://github.com/nmquijada/tormes) | Shell | v1.3.0 | 2021.8 | _Bioinformatics_ | 2019.4 | [TORMES: an automated pipeline for whole bacterial genome analysis.](https://doi.org/10.1093/bioinformatics/btz220)
| [Hybracter](https://github.com/gbouras13/hybracter) | Python | v0.10.0 | 2024.10 | _Microb Genom_ | 2024.5 | [Hybracter: Enabling Scalable, Automated, Complete and Accurate Bacterial Genome Assemblies](https://doi.org/10.1099/mgen.0.001244)


## others
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | -------------------- | ----------- | -------- | ----------- | ----- |
| [KiNext](https://gitlab.ifremer.fr/bioinfo/workflows/kinext) |||||| A pipeline for protein kinase identification, classification and phylogeny
| [BaGPipe](https://github.com/sanger-pathogens/BaGPipe) | Nextflow | v11.0.0 | 2024.5 | bioRxiv | 2025.3 | [BaGPipe: an automated, reproducible, and flexible pipeline for bacterial genome-wide association studies.](https://doi.org/10.1101/2025.02.28.640835)|
