## comparative genomics

### AAI/ANI
- [[ANI calculator](http://enve-omics.ce.gatech.edu/ani/)] - [web]
- [[AAI calculator](http://enve-omics.ce.gatech.edu/aai/)] - [web]
- [[fastANI](https://github.com/ParBLiSS/FastANI)] - [C++, v1.34, 2023.07] - [2018.11, _Nat Comm_] - [[High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries.](https://doi.org/10.1038/s41467-018-07641-9)]
- [[PyFastANI](https://github.com/althonos/pyfastani)] - [Python, v0.6.1, 2025.1] - [2025.2, _bioRxiv_] - [[PyOrthoANI, PyFastANI, and Pyskani: a suite of Python libraries for computation of average nucleotide identity.](https://doi.org/10.1101/2025.02.13.638148)]
- [[PyOrthoANI](https://github.com/althonos/pyorthoani)] - [Python, v0.7.0, 2025.2] - [2025.2, _bioRxiv_] - [[PyOrthoANI, PyFastANI, and Pyskani: a suite of Python libraries for computation of average nucleotide identity.](https://doi.org/10.1101/2025.02.13.638148)]
- [[Pyskani](https://github.com/althonos/pyskani)] - [Python, v0.1.3, 2024.12] - [2025.2, _bioRxiv_] - [[PyOrthoANI, PyFastANI, and Pyskani: a suite of Python libraries for computation of average nucleotide identity.](https://doi.org/10.1101/2025.02.13.638148)]
- [[EzAAI](https://github.com/endixk/ezaai)] - [Java, v1.2.3, 2024.02] - [2021.04, _J Microbiol_] - [[Introducing EzAAI: a pipeline for high throughput calculations of prokaryotic average amino acid identity.](https://doi.org/10.1007/s12275-021-1154-0)]
- [[SynTracker](https://github.com/leylabmpi/SynTracker)] - [Python, v1.3.1, 2024.09 ] - [2024.06, _Nat Biotechnol_] - [[Strain tracking in complex microbiomes using synteny analysis reveals per-species modes of evolution.](https://doi.org/10.1038/s41587-024-02276-2)]
- [[skani](https://github.com/bluenote-1577/skani)] - [Rust, v0.2.2, 2024.07] - [2023.09, _Nat Methods_] - [[Fast and robust metagenomic sequence comparison through sparse chaining with skani](https://doi.org/10.1038/s41592-023-02018-3)]
- [ANIclustermap](https://github.com/moshi4/ANIclustermap) - [Python, v1.4.0, 2024.7] - [Shimoyama, Y. (2022). ANIclustermap: A tool for drawing ANI clustermap between all-vs-all microbial genomes [Computer software].]
- [EvANI](https://github.com/sinamajidian/EvANI) - [Shell/Python, No release] - [2025.2, _bioRxiv_] - [[EvANI benchmarking workflow for evolutionary distance estimation.](https://doi.org/10.1101/2025.02.23.639716)]


### visualisation
- [[gggenes](https://github.com/wilkox/gggenes)] - [Draw gene arrow maps in ggplot2]
- [[LoVis4u](https://github.com/art-egorov/lovis4u)] - [Python, v0.0.11, 2024.10] - [2024.9, _bioRxiv_] - [[LoVis4u: Locus Visualisation tool for comparative genomics.](https://doi.org/10.1101/2024.09.11.612399)]
- [[gggenomes](https://github.com/thackl/gggenomes)] - [gggenomes: A Grammar of Graphics for Comparative Genomics]
- [[plasmapR](https://github.com/BradyAJohnston/plasmapR)] - [Creating plasmid maps inside ggplot.]


### 
- [WAFFLE](https://github.com/biobakery/waafle) - [Python, v1.1.0, 2024.10] - [2025.1, _Nat Microbiol_] - [[Profiling lateral gene transfer events in the human microbiome using WAAFLE.](https://doi.org/10.1038/s41564-024-01881-w)]
- [[SCARAP](https://github.com/swittouck/scarap)] - [Python, v1.0.0, 2024.11] - [2024.12, _Bioinformatics_] - [[SCARAP: scalable cross-species comparative genomics of prokaryotes](https://doi.org/10.1093/bioinformatics/btae735)]


### SV/V
- [[SVbyEye](https://github.com/daewoooo/SVbyEye)] - [R, active dev, No tag] - [2024.09, _bioRxiv_] - [[SVbyEye: A visual tool to characterize structural variation among whole genome assemblies.](https://doi.org/10.1101/2024.09.11.612418)]
- [[CompareM](https://github.com/dparks1134/CompareM)] - [Python, v0.1.2, 2020.12] - [Unsupported. Lasted version Dec 31 2020.]
- [[gcSV](https://github.com/hitbc/gcSV)] - [C,C++, NoRelease] - [2025.2, _bioRxiv_] - [[gcSV: a unified framework for comprehensive structural variant detection.](https://doi.org/10.1101/2025.02.10.637589)] - [human]
- [[minipileup](https://github.com/lh3/minipileup)] - [C, v1.1, 2025.2] - [https://github.com/lh3/minipileup.]