## comparative genomics

### AAI/ANI
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [ANI calculator](http://enve-omics.ce.gatech.edu/ani/) | web
| [AAI calculator](http://enve-omics.ce.gatech.edu/aai/) | web
| [fastANI](https://github.com/ParBLiSS/FastANI) | C++ | v1.34 | 2023.07 | _Nat Comm_ | 2018.11 | [High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries](https://doi.org/10.1038/s41467-018-07641-9)
| [PyFastANI](https://github.com/althonos/pyfastani) | Python | v0.6.1 | 2025.1 | bioRxiv | 2025.2 | [PyOrthoANI, PyFastANI, and Pyskani: a suite of Python libraries for computation of average nucleotide identity](https://doi.org/10.1101/2025.02.13.638148)
| [PyOrthoANI](https://github.com/althonos/pyorthoani) | Python | v0.7.0 | 2025.2 | bioRxiv | 2025.2 | [PyOrthoANI, PyFastANI, and Pyskani: a suite of Python libraries for computation of average nucleotide identity](https://doi.org/10.1101/2025.02.13.638148)
| [Pyskani](https://github.com/althonos/pyskani) | Python | v0.1.3 | 2024.12 | bioRxiv | 2025.2 | [PyOrthoANI, PyFastANI, and Pyskani: a suite of Python libraries for computation of average nucleotide identity](https://doi.org/10.1101/2025.02.13.638148)
| [EzAAI](https://github.com/endixk/ezaai) | Java | v1.2.3 | 2024.02 | _J Microbiol_ | 2021.04 | [Introducing EzAAI: a pipeline for high throughput calculations of prokaryotic average amino acid identity](https://doi.org/10.1007/s12275-021-1154-0)
| [SynTracker](https://github.com/leylabmpi/SynTracker) | Python | v1.3.1 | 2024.09  | _Nat Biotechnol_ | 2024.06 | [Strain tracking in complex microbiomes using synteny analysis reveals per-species modes of evolution](https://doi.org/10.1038/s41587-024-02276-2)
| [skani](https://github.com/bluenote-1577/skani) | Rust | v0.2.2 | 2024.07 | _Nat Methods_ | 2023.09 | [Fast and robust metagenomic sequence comparison through sparse chaining with skani](https://doi.org/10.1038/s41592-023-02018-3)
| [ANIclustermap](https://github.com/moshi4/ANIclustermap) | Python | v1.4.0 | 2024.7 || 2022 | Shimoyama Y. (2022). ANIclustermap: A tool for drawing ANI clustermap between all-vs-all microbial genomes [Computer software]
| [EvANI](https://github.com/sinamajidian/EvANI) | Shell/Python | Norelease || bioRxiv | 2025.2 | [EvANI benchmarking workflow for evolutionary distance estimation](https://doi.org/10.1101/2025.02.23.639716)


### visualisation
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [gggenes](https://github.com/wilkox/gggenes) |||||| Draw gene arrow maps in ggplot2 |
| [LoVis4u](https://github.com/art-egorov/lovis4u) | Python | v0.0.11 | 2024.10 | bioRxiv | 2024.9 | [LoVis4u: Locus Visualisation tool for comparative genomics.](https://doi.org/10.1101/2024.09.11.612399)
| [gggenomes](https://github.com/thackl/gggenomes) |||||| gggenomes: A Grammar of Graphics for Comparative Genomics
| [plasmapR](https://github.com/BradyAJohnston/plasmapR) |||||| Creating plasmid maps inside ggplot
| [geneviewer](https://github.com/nvelden/geneviewer) | R | CRAN release | 2025.1 ||| An R package designed for drawing gene arrow maps


### 
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [WAFFLE](https://github.com/biobakery/waafle) | Python | v1.1.0 | 2024.10 | _Nat Microbiol_ | 2025.1 | [Profiling lateral gene transfer events in the human microbiome using WAAFLE](https://doi.org/10.1038/s41564-024-01881-w)
| [SCARAP](https://github.com/swittouck/scarap) | Python | v1.0.0 | 2024.11 | _Bioinformatics_ | 2024.12 | [SCARAP: scalable cross-species comparative genomics of prokaryotes](https://doi.org/10.1093/bioinformatics/btae735)


### SV/V
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [SVbyEye](https://github.com/daewoooo/SVbyEye) | R ||| bioRxiv | 2024.09 | [SVbyEye: A visual tool to characterize structural variation among whole genome assemblies](https://doi.org/10.1101/2024.09.11.612418) | active dev
| [CompareM](https://github.com/dparks1134/CompareM) | Python | v0.1.2 | 2020.12 ||| Unsupported. Lasted version Dec 31 2020
| [gcSV](https://github.com/hitbc/gcSV) | C/C++ ||| bioRxiv | 2025.2 | [gcSV: a unified framework for comprehensive structural variant detection](https://doi.org/10.1101/2025.02.10.637589) | human
| [minipileup](https://github.com/lh3/minipileup) | C | v1.1 | 2025.2 ||| Minipileup is a simple pileup-based variant caller. It takes a reference FASTA and one or multiple alignment BAM as input, and outputs a multi-sample VCF along with allele counts |
