## phylogeny
### build a tree
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [IQ-TREE](http://www.iqtree.org) | Java | v2.3.6 | 2024.8 | _Mol Biol Evol_ | 2020.2 | [IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era](https://doi.org/10.1093/molbev/msaa015) |
| [iqtree -m](http://www.iqtree.org/#download) | Java | v2.3.6 | 2024.8 | _Nat Methods_ | 2017.5 | [ModelFinder: fast model selection for accurate phylogenetic estimates.](https://doi.org/10.1038/nmeth.4285) |
| [fasttree2](http://www.microbesonline.org/fasttree/) | C++ | v2.1.11 || _PloS ONE_ | 2010.03 | [FastTree 2 -- Approximately Maximum-Likelihood Trees for Large Alignments](https://doi.org/10.1371/journal.pone.0009490) |
| [PhyloPhlAn](https://github.com/biobakery/phylophlan) | Python | v3.1.1 | 2024.03 | _Nat Commun_ | 2020.05 | [Precise phylogenetic analysis of microbial isolates and genomes from metagenomes using PhyloPhlAn 3.0](https://doi.org/10.1038/s41467-020-16366-7)
| [MrBayes](https://github.com/NBISweden/MrBayes) | C ||||| [MRBAYES: Bayesian inference of phylogenetic trees](https://doi.org/10.1093/bioinformatics/17.8.754)
| [MrBayes](https://github.com/NBISweden/MrBayes) | C | v3 || _Bioinformatics_ | 2003.8 | [MrBayes 3: Bayesian phylogenetic inference under mixed models](https://doi.org/10.1093/bioinformatics/btg180)
| [MrBayes](https://github.com/NBISweden/MrBayes) | C | v3.2.7 | 2020.1 | _Syst Biol_ | 2012.2 | [MRBAYES 3.2: Efficient Bayesian phylogenetic inference and model selection across a large model space](https://doi.org/10.1093/sysbio/sys029)
| [trimAI](https://github.com/inab/trimal) | C++ | v1.5.0 | 2024.7 | _Bioinformatics_ | 2009.8 | [trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses](https://doi.org/10.1093/bioinformatics/btp348)
| [RAxML-NG](https://github.com/amkozlov/raxml-ng) | C++ | v1.2.2 | 2024.5 | _Bioinformatics_ | 2019.11 | [RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference](https://doi.org/10.1093/bioinformatics/btz305)
| [getphylo](https://github.com/drboothtj/getphylo) | Python ||| _BMC Bioinform_ | 2025.1 | [getphylo: rapid and automatic generation of multi-locus phylogenetic trees.](https://doi.org/10.1186/s12859-025-06035-1)
| [Poplar](https://github.com/sandialabs/poplar) | Python ||| bioRxiv | 2024.11 | [Poplar: A Phylogenetics Pipeline](https://doi.org/10.1101/2024.11.11.623070)


### view the tree
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [iTol](https://itol.embl.de) | Web ||| _Nucleic Acids Res_ | 2024.04 | [Interactive Tree of Life (iTOL) v6: recent updates to the phylogenetic tree display and annotation tool](https://doi.org/10.1093/nar/gkae268)
| [ARB](http://www.arb-home.de) | 需要转发 | arb-7.0 | 2021.09 | _Nucleic Acids Res_ | 2004.03 | [ARB: a software environment for sequence data.](https://doi.org/10.1093/nar/gkh293)
| [FigTree](https://github.com/rambaut/figtree/) | Java | v1.4.5-pre | 2024.02
| [GraPhlAn](https://github.com/biobakery/graphlan) | Python | 1.1.3 | 2020.06 | _PeerJ_ | 2015.06 | [Compact graphical representation of phylogenetic data and metadata with GraPhlAn](https://doi.org/10.7717/peerj.1029)
| [phyTreeViz](https://github.com/moshi4/phyTreeViz) | Python | v0.2.0 | 2024.1 || 2023 | Shimoyama Y. (2023). phyTreeViz: Simple phylogenetic tree visualization python package [Computer software]
