## microbial profile, diversity, network, and interactions

### abundance / profile
- [[coverM](https://github.com/wwood/CoverM)] - [Rust, v0.7.0, 2024.01] -  [fast DNA read coverage and relative abundance calculator focused on metagenomics applications.] - [2025.1, _arXiv_] - [CoverM: Read alignment statistics for metagenomics](https://doi.org/10.48550/arXiv.2501.11217)
- [[Fairy](https://github.com/bluenote-1577/fairy)] - [Rust, v0.5.5, 2024.05] - [2024.08, _Microbiome_] - [[Fairy: fast approximate coverage for multi-sample metagenomic binning.](https://doi.org/10.1186/s40168-024-01861-6)]
- [[Tronko](https://github.com/lpipes/tronko)] - [C, NoRelese] - [[A rapid phylogeny-based method for accurate community profiling of large-scale metabarcoding datasets.](https://doi.org/10.7554/eLife.85794)]
- [[MUSET](https://github.com/CamilaDuitama/muset)] - [C++, v0.5.1, 2024.12] - [2025.2, _Bioinformatics_] - [[MUSET: Set of utilities for constructing abundance unitig matrices from sequencing data.](https://doi.org/10.1093/bioinformatics/btaf054)]


### diversity
- [[q2-kmerizer](https://github.com/bokulich-lab/q2-kmerizer)] - [2025.2, _mSystems_] - [[Integrating sequence composition information into microbial diversity analyses with k-mer frequency counting.](https://doi.org/10.1128/msystems.01550-24)]

### network
- [SpeSpeNet](https://tbb.bio.uu.nl/SpeSpeNet) - [Web, v1.0] - [2025.2, _ISME Communications_] - [[SpeSpeNet: an interactive and user-friendly tool to create and explore microbial correlation networks.](https://doi.org/10.1093/ismeco/ycaf036)]


### interactions / associations
- [[PHLAME](https://github.com/quevan/phlame)] - [Python, NoRelease] - [2025.2, _bioRxiv_] - [[Intraspecies associations from strain-rich metagenome samples.](https://doi.org/10.1101/2025.02.07.636498)]

- [[Prediction of strain level phageâ€“host interactions across the Escherichia genus using only genomic information.](https://doi.org/10.1038/s41564-024-01832-5)] - [2024.10, _Nat Microbiol_] - [code and  files](https://github.com/mdmparis/coli_phage_interactions_2023) - [[Viral host Range database](https://viralhostrangedb.pasteur.cloud/.Source data)]

- [[Prediction of Klebsiella phage-host specificity at the strain level.](https://doi.org/10.1038/s41467-024-48675-6)] - [2024.5, _Nat Commun_] - [[code and files](https://github.com/dimiboeckaerts/PhageHostLearn)]

- [iPHoP](https://bitbucket.org/srouxjgi/iphop/)] - [*, v1.3.3, *] - [2023.4, _PLoS Biol_] - [[iPHoP: An integrated machine learning framework to maximize host prediction for metagenome-derived viruses of archaea and bacteria](https://doi.org/10.1371/journal.pbio.3002083)]