## metatranscriptomics
### meta RNA quantification
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [sailfish](https://github.com/kingsfordgroup/sailfish) | C++ | v0.10.0 | 2016.4 | _Nat Biotechnol_ | 2014.4 | [Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms](https://doi.org/10.1038/nbt.2862)
| [SortMeRNA](https://github.com/sortmerna/sortmerna) | C++/C | v4.3.7 | 2024.04 | _Bioinformatics_ | 2012.10 | [SortMeRNA: Fast and accurate filtering of ribosomal RNAs in metatranscriptomic data](https://doi.org/10.1093/bioinformatics/bts611)
| [salmon](https://github.com/COMBINE-lab/salmon) | C++ | v1.10.1 | 2023.03 | _Nat Methods_ | 2017.03 | [Salmon provides fast and bias-aware quantification of transcript expression](https://doi.org/10.1038/nmeth.4197)
| [RiboDetector](https://github.com/hzi-bifo/RiboDetector) | Python | v0.3.1 | 2024.1 | _Nucleic Acids Res_ | 2022.6 | [Rapid and accurate identification of ribosomal RNA sequences via deep learning](https://doi.org/10.1093/nar/gkac112)
| [kallisto](https://github.com/pachterlab/kallisto) | C/C++ | v0.51.1 | 2024.9 | _Nat Biotechnol_ | 2016.4 | [Near optimal probabilistic RNA-seq quantification](https://doi.org/10.1038/nbt.3519)
| [NanoCount](https://github.com/a-slide/NanoCount) | Python | v1.1.0 | 2022.12 | _Nucleic Acids Res_ | 2022.2 | [Accurate expression quantification from nanopore direct RNA sequencing with NanoCount](https://doi.org/10.1093/nar/gkab1129)
| [step by step shell and python code](https://github.com/RyoMameda/workflow) |||| bioRxiv | 2025.1 | [Data-driven workflow for comprehensive gene expression analysis in complex](https://doi.org/10.1101/2025.01.17.632662)


## assembly
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [SPAdes/rnaSPAdes](https://github.com/ablab/spades) | C++ | v4.0.0 | 2024.6 | _Gigascience_ | 2019.9 | [rnaSPAdes: a de novo transcriptome assembler and its application to RNA-Seq data](https://doi.org/10.1093/gigascience/giz100)
| [idba/IDBA-tran](https://github.com/loneknightpy/idba) | C++ | v1.1.3 | 2016.7 | _Bioinformatics_ | 2013.7 | [IDBA-tran: a more robust de novo de Bruijn graph assembler for transcriptomes with uneven expression levels](https://doi.org/10.1093/bioinformatics/btt219)

## sv
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [Clair3-RNA](https://github.com/HKU-BAL/Clair3-RNA) | Python | 2025.1 | v0.2.1 | bioRxiv | 2024.11 | [Clair3-RNA: A deep learning-based small variant caller for long-read RNA sequencing data](https://doi.org/10.1101/2024.11.17.624050)

## stats
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [GraphPCA](https://github.com/YANG-ERA/GraphPCA) | Python | GraphPCA-746ef16 | 2024.8 | _Genome Biol_ | 2024.11 | [GraphPCA: a fast and interpretable dimension reduction algorithm for spatial transcriptomics data](https://doi.org/10.1186/s13059-024-03429-x)