## genes and elements prediction
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [Prodigal](https://github.com/hyattpd/Prodigal) | C | v2.6.3 | 2016.02 | _BMC Bioinformatics_ | 2010.03 | [Prodigal: prokaryotic gene recognition and translation initiation site identification.](https://doi.org/10.1186/1471-2105-11-119)
| [Pyrodigal](https://github.com/althonos/pyrodigal) | Cython/Python | v3.5.2 | 2024.9 | _J Open Source Softw_ | 2022.4 | [Pyrodigal: Python bindings and interface to Prodigal, an efficient method for gene prediction in prokaryotes](https://joss.theoj.org/papers/10.21105/joss.04296)
| [Prokka](https://github.com/tseemann/prokka) | Perl | v1.14.5 | 2019.11 | _Bioinformatics_ | 2014.03 | [Prokka: rapid prokaryotic genome annotation](https://doi.org/10.1093/bioinformatics/btu153)
| [Barrnap](https://github.com/tseemann/barrnap) | Perl/Shell | 0.9 | 2018.4 ||| BAsic Rapid Ribosomal RNA Predictor. Barrnap predicts the location of ribosomal RNA genes in genomes
| [RNAmmer](https://services.healthtech.dtu.dk/services/RNAmmer-1.2/) || v1.2 |
| [tRNAscan-SE 2.0](https://github.com/UCSC-LoweLab/tRNAscan-SE) | C/Perl | v2.0.12 | 2022.11 | _Nucleic Acids Res_ | 2021.8 | [tRNAscan-SE 2.0: improved detection and functional classification of transfer RNA genes](https://doi.org/10.1093/nar/gkab688)
| [tRNAscan-SE](https://trna.ucsc.edu/tRNAscan-SE/) | Web ||| _Methods Mol Biol_ | 2019.5 | [tRNAscan-SE: Searching for tRNA Genes in Genomic Sequences](https://doi.org/10.1007/978-1-4939-9173-0_1)
| [PILERCR](http://www.drive5.com/pilercr/) || v1.06 || _BMC Bioinform_ | 2007.1 | [PILER-CR: Fast and accurate identification of CRISPR repeats.](https://doi.org/10.1186/1471-2105-8-18)
| [pybarrnap](https://github.com/moshi4/pybarrnap) | Python | v0.5.0 | 2024.3 || 2024 | pybarrnap: Python implementation of barrnap [Computer software]


## small moleculars | peptides
### signal peptides
| Tool/workflows | Language | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | -------- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [SignalP](https://github.com/fteufel/signalp-6.0) | Python | v6 | 2022.1 | _Nat Biotechnol_ | 2022.1 | [SignalP 6.0 predicts all five types of signal peptides using protein language models](https://doi.org/10.1038/s41587-021-01156-3)
| [DeepLocPro](https://github.com/Jaimomar99/deeplocpro) |||| _Bioinformatics_ |  2024.11 | [Predicting the subcellular location of prokaryotic proteins with DeepLocPro](https://doi.org/10.1093/bioinformatics/btae677) | [web api](https://ku.biolib.com/deeplocpro)


## databases
| Tool/workflows | Type | Lastest version | Release date | Journal | Pub Date | Paper title | Notes |
| -------------- | ---- | --------------- | ------------ | ------- | -------- | ----------- | ----- |
| [PIPdb](https://nmdc.cn/pipdb) | plasmids in pathogens || 2024.9 | _Nucleic Acids Res_ | 2024.10 | [PIPdb: a comprehensive plasmid sequence resource for tracking the horizontal transfer of pathogenic factors and antimicrobial resistance genes](https://doi.org/10.1093/nar/gkae952) ｜
| [PlasmidScope](https://plasmid.deepomics.org/) | Website ||| _Nucleic Acids Res_ | 2024.10 | [PlasmidScope: a comprehensive plasmid database with rich annotations and online analytical tools.](https://doi.org/10.1093/nar/gkae930) | download database ｜
