## annotation

### tools
- [[eggNOG-mapper v2](https://github.com/eggnogdb/eggnog-mapper)] - [Python, v2.1.12, 2023.08] - [2021.10, _Mol Biol Evol_]- [[eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale.](https://doi.org/10.1093/molbev/msab293)]
- [[KofamScan](https://www.genome.jp/ftp/tools/kofam_scan/)] - [Ruby, v1.3.0, 2020.05] - [2019.11, _Bioinformatics_] - [[KofamKOALA: KEGG ortholog assignment based on profile HMM and adaptive score threshold.](https://doi.org/10.1093/bioinformatics/btz859)]
- [[Macrel](https://github.com/BigDataBiology/macrel)] - [Python, v1.5.0, 2024.09] - [2020.12, _PeerJ_] - [[Macrel: antimicrobial peptide screening in genomes and metagenomes.](https://doi.org/10.7717/peerj.10555)]
- [[RGI](https://github.com/arpcard/rgi)] - [Python, 6.0.3, 2023.09] - [2023.01, _Nucleic Acids Res_] - [[CARD 2023: expanded curation, support for machine learning, and resistome prediction at the Comprehensive Antibiotic Resistance Database.](https://doi.org/10.1093/nar/gkac920)]
- [[MetaCerberus](https://github.com/raw-lab/metacerberus)] - [Python, v1.4.0, 2024.08] - [2024.02, _Bioinformatics_] - [[MetaCerberus: distributed highly parallelized HMM-based processing for robust functional annotation across the tree of life.](https://doi.org/10.1093/bioinformatics/btae119)]
- [[GMSC-mapper](https://github.com/BigDataBiology/GMSC-mapper)] - [Python, v0.1.0, 2024.04] - [2024.08, _Nat Commun_] - [[A catalog of small proteins from the global microbiome.](https://doi.org/10.1038/s41467-024-51894-6)]
- [[Bakta](https://github.com/oschwengers/bakta)] - [Python, v1.10.1, 2024.11] - [2021.11, _Microbial Genomics_] [Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification.](https://doi.org/10.1099/mgen.0.000685) - [[web version](https://bakta.computational.bio)]
- [[DRAM](https://github.com/WrightonLabCSU/DRAM)] - [Python, v1.5.0, 2024.1] - [2020.9, _Nucleic Acids Res_] - [[DRAM for distilling microbial metabolism to automate the curation of microbiome function](https://doi.org/10.1093/nar/gkaa621)]
- [[graftM](https://github.com/geronimp/graftM)] - [Python, v0.14.0, 2022.5] - [2018.6, _Nucleic Acids Res_] - [[GraftM: a tool for scalable, phylogenetically informed classification of genes within metagenomes.](https://doi.org/10.1093/nar/gky174)]
- [CD-Search](http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi) - [Web] - [2004.7, _Nucleic Acids Res_] - [CD-Search: protein domain annotations on the fly.](https://doi.org/10.1093/nar/gkh454)
- [[HMMER3](https://github.com/EddyRivasLab/hmmer)] - [C, v3.4, 2023.8] - [2011.10, _PLoS Comput Biol_] - [[Accelerated profile HMM searches.](https://doi.org/10.1371/journal.pcbi.1002195)]
- [[DeepARG](https://github.com/gaarangoa/deeparg)] - [Python2.7, NA, 2023.11] - [2018.2, _Microbiome_] - [[DeepARG: a deep learning approach for predicting antibiotic resistance genes from metagenomic data](https://doi.org/10.1186/s40168-018-0401-z)]
- [ARG-OAP](https://galaxyproject.org/use/args-oap/)
- [[argNorm](https://github.com/BigDataBiology/argNorm)] - [Python, v0.6.0, 2024.8] - [2024.10, _QUT ePrints_] - [[argNorm: Normalization of antibiotic resistance gene annotations to the Antibiotic Resistance Ontology (ARO)](https://eprints.qut.edu.au/252448/)]
- [[OrthoLoger](https://orthologer.ezlab.org)] - [bash, v3.5.0, 2024.10] - [2022.11, _Nucleic Acids Res_] - [[OrthoDB v11: annotation of orthologs in the widest sampling of organismal diversity](https://doi.org/10.1093/nar/gkac998)]
- [[nail](https://github.com/TravisWheelerLab/nail)] - [Rust, 2024.7, v0.2.0] - [2024.1, _bioRxiv_] - [[nail: software for high-speed, high-sensitivity protein sequence annotation](https://doi.org/10.1101/2024.01.27.577580)]
- [[FastOMA](https://github.com/DessimozLab/FastOMA/)] - [Python, 2024.9,v0.3.4] - [2025.1, _Nat Methods_] - [[Orthology inference at scale with FastOMA.](https://doi.org/10.1038/s41592-024-02552-8)]

### databases
- [[eggNOG 6.0](http://eggnog6.embl.de/)] - [AA Genes, v6.0, 2022.09] - [2022.11, _Nucleic Acids Res_] - [[eggNOG 6.0: Enabling comparative genomics across 12,535 organisms.](https://doi.org/10.1093/nar/gkac1022)]
- [[KEGG](https://www.genome.jp/kegg/)] - [AA Genes, 111.0, 2024.08] - [2016.1, _Nucleic Acids Res_] - [[KEGG as a reference resource for gene and protein annotation](https://doi.org/10.1093/nar/gkv1070)]
- [[CAZy](http://www.cazy.org)] - [AA, web] - [2022.1, _Nucleic Acids Res_] - [[The carbohydrate-active enzyme database: functions and literature.](https://doi.org/10.1093/nar/gkab1045)]
- [[CARD](https://card.mcmaster.ca)] - [ARG, v3.3.0, 2024.08] - [2023.01, _Nucleic Acids Res_] - [[CARD 2023: expanded curation, support for machine learning, and resistome prediction at the Comprehensive Antibiotic Resistance Database.](https://doi.org/10.1093/nar/gkac920)]
- [[UniProt](https://www.uniprot.org)] - [AA genes, v2024_04, 2024.07] - [2022.11, _Nucleic Acids Res_] - [[The UniProt Consortium. UniProt: the Universal Protein Knowledgebase in 2023.](https://doi.org/10.1093/nar/gkac1052)]
- [[UniProt](https://www.uniprot.org/)] - [AA genes, v2024_5] - [2024.11, _Nucleic Acids Res_] - [UniProt: the Universal Protein Knowledgebase in 2025.](https://doi.org/10.1093/nar/gkae1010)
- [[ISOSDB](https://github.com/joshuakirsch/pseudoR)] - [IS Sequences, v3, 2023.12] - [2024.05, _Cell Host Microbe_] - [[A metagenomics pipeline reveals insertion sequence-driven evolution of the microbiota.](https://doi.org/10.1016/j.chom.2024.03.005)]
- [[AMPSphere](https://ampsphere.big-data-biology.org/home)] - [AMP, v2022-03, 2022.03] - [2024.07, _cell_] - [[Discovery of antimicrobial peptides in the global microbiome with machine learning.](https://doi.org/10.1016/j.cell.2024.05.013)]
- [[DRAMP](http://dramp.cpu-bioinfor.org)] - [AMP, v4.0, 2024.09] - [2022.01, _Nucleic Acids Res_] - [[DRAMP 3.0: an enhanced comprehensive data repository of antimicrobial peptides.](https://doi.org/10.1093/nar/gkab651)]
- [[GMSC](https://gmsc.big-data-biology.org)] - [smORF, v1.0, 2024.08] - [2024.08, _Nat Commun_] - [[A catalog of small proteins from the global microbiome.](https://doi.org/10.1038/s41467-024-51894-6)]
- [[TCDB](https://www.tcdb.org)] - [transporters, 2024-09, 2024.09] - [2021.01, _Nucleic Acids Res_] - [[The Transporter Classification Database (TCDB): 2021 update](https://doi.org/10.1093/nar/gkaa1004)]
- [[VFDB](http://www.mgc.ac.cn/VFs/main.htm)] - [virulence factors, 2024.9] - [2022.1, _Nucleic Acids Res_] - [VFDB 2022: a general classification scheme for bacterial virulence factors](https://doi.org/10.1093/nar/gkab1107)
- [[DoriC](https://tubic.org/doric/home)] - [oriC, v12.1, NA] - [2023.1, _Nucleic Acids Res_] - [[DoriC 12.0: an updated database of replication origins in both complete and draft prokaryotic genomes.](https://doi.org/10.1093/nar/gkac964)]
- [[TIGRAFMs](http://tigrfams.jcvi.org/cgi-bin/index.cgi)] - [protein sequence, v15.0, 2014.9] [2001.1, _Nucleic Acids Res_] - [TIGRFAMs: a protein family resource for the functional identification of proteins.](https://doi.org/10.1093/nar/29.1.41)
- [[Pfam](https://www.ebi.ac.uk/interpro/download/Pfam/)] - [protein sequences, v37.0, 2024.5] - [2021.1, _Nucleic Acids Res_] - [[Pfam: The protein families database in 2021.](https://doi.org/10.1093/nar/gkaa913)]
- [[Rfam](https://rfam.org)] - [microRNA, v15, 2024.9] - [2021.1, _Nucleic Acids Res_] - [[Rfam 14: expanded coverage of metagenomic, viral and microRNA families.](https://doi.org/10.1093/nar/gkaa1047)]
- [[PHI](https://phi5.phi-base.org)] - [Pathogen Host Interactions, v5.0, 2024.3] [2022.1, _Nucleic Acids Res_] [[PHI-base in 2022: a multi-species phenotype database for Pathogen–Host Interactions.](https://doi.org/10.1093/nar/gkab1037)]
- [[oriTDB](https://bioinfo-mml.sjtu.edu.cn/oriTDB2/)] - [2024.10, _Nucleic Acids Res_] - [[oriTDB: a database of the origin-of-transfer regions of bacterial mobile genetic elements.](https://doi.org/10.1093/nar/gkae869)]
- [[COG](https://www.ncbi.nlm.nih.gov/research/COG)] - [2024.11, _Nucleic Acids Res_] - [[COG database update 2024.](https://doi.org/10.1093/nar/gkae983)] - [[FTP](https://ftp.ncbi.nlm.nih.gov/pub/COG/)]
- [[OrthoDB](https://www.orthodb.org)] - [v11.0] - [2022.11, _Nucleic Acids Res_] - [[OrthoDB v11: annotation of orthologs in the widest sampling of organismal diversity](https://doi.org/10.1093/nar/gkac998)]
- [CAZyme3D](https://pro.unl.edu/CAZyme3D/) - [web] - [2024.12, _bioRxiv_] - [[CAZyme3D: a database of 3D structures for carbohydrate-active enzymes](https://doi.org/10.1101/2024.12.27.630555)]
- [S9BactDB](http://caps.ncbs.res.in/S9BactDB) - [S9 proteases, web] - [2025.1, _bioRxiv_] - [[S9BactDB: A database for S9 family of proteases in bacterial genomes.](https://doi.org/10.1101/2025.01.01.631042)]

### protein structure
- [[Chai-1](https://github.com/chaidiscovery/chai-lab)] - [Python, v0.0.1, 2024.09] - [2024.09, Web] - [[Introducing Chai-1: Decoding the molecular interactions of life.](https://www.chaidiscovery.com/blog/introducing-chai-1)]
- [[AlphaFold2](https://github.com/google-deepmind/alphafold)] - [Python, v2.3.2, 2023.04] - [2021.07, _Nature_] - [[Highly accurate protein structure prediction with AlphaFold.](https://doi.org/10.1038/s41586-021-03819-2)]
- [[AlphaFold3](https://www.alphafoldserver.com/)] - [Web Service, NA, NA] - [2024.05, _Nature_] - [[Accurate structure prediction of biomolecular interactions with AlphaFold 3.](https://doi.org/10.1038/s41586-024-07487-w)]
- [[DNAproDB](https://dnaprodb.usc.edu/)] - [Online database] - [2024.11, _Nucleic Acids Res_] - [[DNAproDB: an updated database for the automated and interactive analysis of protein–DNA complexes.](https://doi.org/10.1093/nar/gkae970)]
