## 组装 | assembly

此列表收集了与组装相关的工具，包括短读长、长度长和混合组装。

### 组装 - 短读长 | short-reads
- [[SOAPdenovo2](https://github.com/aquaskyline/SOAPdenovo2)] - [C, r242, 2020.10] - [2012.12, _GigaSci_] - [[SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.](https://doi.org/10.1186/2047-217X-1-18)]
- [[idba/IDBA-UD](https://github.com/loneknightpy/idba)] - [C++, v1.1.3, 2016.07] - [2012.04, _Bioinformatics_] - [[IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth](https://doi.org/10.1093/bioinformatics/bts174)]
- [[SPAdes/metaSPAdes](https://github.com/ablab/spades)] - [C++, v4.0.0, 2024.06] - [2017.05, _Genome Res_] - [[metaSPAdes: a new versatile metagenomic assembler.](https://doi.org/10.1101/gr.213959.116)]
- [[megahit](https://github.com/voutcn/megahit)] - [C++, v0.3.3-a, 2015.7] - [2015.01, _Bioinformatics_] - [[MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.](https://doi.org/10.1093/bioinformatics/btv033)]
- [[megahit](https://github.com/voutcn/megahit)] - [C++, v1.2.9, 2019.10] - [2016.6, _Methods_] -[[MEGAHIT v1.0: A Fast and Scalable Metagenome Assembler driven by Advanced Methodologies and Community Practices.](https://doi.org/10.1016/j.ymeth.2016.02.020)]
- [[PenuiN](https://github.com/soedinglab/plass)] - [C, v5-cf8933, 2024.03] - [2024.08, _bioRxiv_] - [[Strain-resolved de-novo metagenomic assembly of viral genomes and microbial 16S rRNAs.](https://doi.org/10.1101/2024.03.29.587318)]
- [[BBMerge](https://sourceforge.net/projects/bbmap/)] - [Bash, v39.10, 2024.9] - [2017.10, _PLoS One_] - [[BBMerge - Accurate paired shotgun read merging via overlap.](https://doi.org/10.1371/journal.pone.0185056)]
- [[FLASH](https://github.com/ebiggers/flash)] - [C, v1.2.11, 2018.3] - [2011.9, _Bioinformatics_] - [FLASH: fast length adjustment of short reads to improve genome assemblies.](https://doi.org/10.1093/bioinformatics/btr507)
- [[ABySS](https://github.com/bcgsc/abyss)] - [C++/C, v2.3.9, 2024.9] - [2017.2, _Genome Res_] - [[ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter.](http://www.genome.org/cgi/doi/10.1101/gr.214346.116)]


### 组装 - 长读长 | long-reads
- [[MaSuRCA](https://github.com/alekseyzimin/masurca)] - [M4/Perl, v4.1.1, 2024.2] - [2013.8, _Bioinformatics_] - [[The MaSuRCA genome assembler.](https://doi.org/10.1371/journal.pcbi.1009860)]
- [[metaFlye](https://github.com/fenderglass/Flye)] - [C/C++, v2.9.5, 2024.08] - [2020.10, _Nat Methods_] - [[metaFlye: scalable long-read metagenome assembly using repeat graphs.](https://doi.org/10.1038/s41592-020-00971-x)]
- [[Canu](https://github.com/marbl/canu)] - [C++/C, v2.2, 2021.08] - [2017.03, _Genome Res_] - [[Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.](https://genome.cshlp.org/content/27/5/722)]
- [[FALCON](https://github.com/PacificBiosciences/FALCON)] - [Python, v0.3.0, 2018.08]
- [[miniasm](https://github.com/lh3/miniasm)] - [TeX/C, v0.3, 2018.07] - [2016.03, _Bioinformatics_] - [[Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences.](https://doi.org/10.1093/bioinformatics/btw152)]
- [[wtdbg2](https://github.com/ruanjue/wtdbg2)] - [C, v2.5, 2019.09] - [2019.12, _Nat Methods_] - [[Fast and accurate long-read assembly with wtdbg2.](https://doi.org/10.1038/s41592-019-0669-3)]
- [[Trycycler](https://github.com/rrwick/Trycycler)] - [Python, v0.5.5, 2024.03] - [2021.09, _Genome Biol_] - [[Trycycler: consensus long-read assemblies for bacterial genomes.](https://doi.org/10.1186/s13059-021-02483-z)]
- [[NextDenovo](https://github.com/Nextomics/NextDenovo)] - [C, 2.5.2, 2023.03] - [2024.04, _Genome Biol_] - [[NextDenovo: an efficient error correction and accurate assembly tool for noisy long reads.](https://doi.org/10.1186/s13059-024-03252-4)]

### 组装 - 混合 | hybrid
- [[idba/IDBA-Hybrid](https://github.com/loneknightpy/idba)] - [C++, v1.1.3, 2016.07] - [2012.04, _Bioinformatics_] - [[IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth](https://doi.org/10.1093/bioinformatics/bts174)]
- [[SPAdes/metaSPAdes](https://github.com/ablab/spades)] - [C++, v4.0.0, 2024.06] - [2017.05, _Genome Res_] - [[metaSPAdes: a new versatile metagenomic assembler.](https://doi.org/10.1101/gr.213959.116)]
- [[Aviary](https://github.com/rhysnewell/aviary)] - [Python, v0.9.2, 2024.09] - [A hybrid assembly and MAG recovery pipeline (and more!)]
- [[OPERA-MS](https://github.com/CSB5/OPERA-MS)] - [C++/Python, v0.8.3, 2020.06] - [2019.07, _Nat Biotechnol_] - [[Hybrid metagenomic assembly enables high-resolution analysis of resistance determinants and mobile elements in human microbiomes.](https://doi.org/10.1038/s41587-019-0191-2)]
- [[Unicycler](https://github.com/rrwick/Unicycler)] - [C++, v0.5.1, 2024.07] - [2017.06, _PloS Comput Biol_] - [[Unicycler: resolving bacterial genome assemblies from short and long sequencing reads.](https://doi.org/10.1371/journal.pcbi.1005595)]
- [[MaSuRCA](https://github.com/alekseyzimin/masurca)] - [M4/Perl, v4.1.1, 2024.2] - [2017.5, _Genome Res_] - [[Hybrid assembly of the large and highly repetitive genome of Aegilops tauschii, a progenitor of bread wheat, with the mega-reads algorithm.](https://doi.org/10.1101/gr.213405.116)]


## PB HiFi
- [[hifiasm-meta](https://github.com/xfengnefx/hifiasm-meta/)] - [C++, v0.3.2, 2024.9] - [2024.04 _Genome Biol_] - [[Evaluating and improving the representation of bacterial contents in long-read metagenome assemblies.](https://doi.org/10.1186/s13059-024-03234-6)]
- [[metaMDBG](https://github.com/GaetanBenoitDev/metaMDBG)] - [C++, v1.0, 2024.8] - [2024.1, _Nat Biotechnol_] - [[High-quality metagenome assembly from long accurate reads with metaMDBG.](https://doi.org/10.1038/s41587-023-01983-6)]

### 抛光 | polish
- [[FMLRC2](https://github.com/HudsonAlpha/fmlrc2)] - [Rust, v0.1.8, 2022.07] - [2023.03, _Mol Bio Evo_] - [[Polishing de novo nanopore assemblies of bacteria and eukaryotes with FMLRC2.](https://doi.org/10.1093/molbev/msad048)]
- [[pilon](https://github.com/broadinstitute/pilon)] - [Scala, v1.24, 2021.01] - [2014.11, _Plos ONE_] - [[Pilon: An Integrated Tool for Comprehensive Microbial Variant Detection and Genome Assembly Improvement.](https://doi.org/10.1371/journal.pone.0112963)]
- [[NextPolish](https://github.com/Nextomics/NextPolish)] - [Rust, v1.4.1, 2022.07] - [2019.11, _Bioinformatics_] - [[NextPolish: a fast and efficient genome polishing tool for long read assembly.](https://doi.org/10.1093/bioinformatics/btz891)]
- [[Polypolish](https://github.com/rrwick/Polypolish)] - [Rust, v0.6.0, 2024.01] - [2024.06, _Microbial Genomics_] - [[How low can you go? Short-read polishing of Oxford Nanopore bacterial genome assemblies.](https://doi.org/10.1099/mgen.0.001254)]
- [[pypolca](https://github.com/gbouras13/pypolca)] - [Python, v0.3.1, 2024.02] - [2024.06, _Microbial Genomics_] - [[How low can you go? Short-read polishing of Oxford Nanopore bacterial genome assemblies.](https://doi.org/10.1099/mgen.0.001254)]
- [MaSuRCA/POLCA](https://github.com/alekseyzimin/masurca) - [M4/Perl, v4.1.1, 2024.2] - [2020.6, _PLoS Comput Biol_] - [[The genome polishing tool POLCA makes fast and accurate corrections in genome assemblies.](https://doi.org/10.1371/journal.pcbi.1007981)]
- [Racon](https://github.com/lbcb-sci/racon) - [C++, v1.5.0, 2021.12] - [2017.1, _Genome Res_] - [Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.](http://www.genome.org/cgi/doi/10.1101/gr.214270.116)

Please cite SAMBA as follows: Zimin AV, Salzberg SL. The SAMBA tool uses long reads to improve the contiguity of genome assemblies. PLoS computational biology. 2022 Feb 4;18(2):e1009860.
