## benchmarks and tutorials

## benchmarks
- [[A realistic benchmark for differential abundance testing and confounder adjustment in human microbiome studies.](https://doi.org/10.1186/s13059-024-03390-9)] - [2024.9, _Genome Biol_]
- [[Removal of sequencing adapter contamination improves microbial genome databases.](https://doi.org/10.1186/s12864-024-10956-1)] - [2024.11, _BMC Genomics_] - [[codes and fiels repo](https://github.com/CUMoellerLab/MalAdapter)]
- [[Comprehensive benchmarking of tools for nanopore-based detection of DNA methylation.](https://doi.org/10.1101/2024.11.09.622763)] - [2024.11, _bioRxiv_]

- [[Assessing the performance of current strain resolution tools on long-read metagenomes.](https://doi.org/10.1101/2024.11.20.624313)] - [2024.11, _bioRxiv_]
- [[A systematic assessment of phylogenomic approaches for microbial species tree reconstruction.](https://doi.org/10.1101/2024.11.20.624597)] - [2024.11, _bioRxiv_]
- [[Benchmarking the impact of reference genome selection on taxonomic profiling accuracy.](https://doi.org/10.1101/2025.02.07.637076)] - [2025.2, _bioRxiv_] - [[codes and tables repo](https://github.com/JaspervB-tud/reference_set_selection_benchmark)]
- [[Are reads required? High-precision variant calling from bacterial genome assemblies](https://doi.org/10.1101/2025.02.20.639404)] - [2025.2, _bioRxiv_] - [[code, tables repo](https://github.com/rrwick/Are-reads-required)]
- [[nf-core/variantbenchmarking v1.0.0: Nextflow pipeline to evaluate and validate the accuracy of variant calling methods](https://github.com/nf-core/variantbenchmarking)] - [Nextflow, v1.0.0, 2025.2]
- [[Cutting edge deep-learning based tools for metagenomic research](https://doi.org/10.1093/nsr/nwaf056)] - [2025.2, _National Science Review_] - [We explore 12 deep-learning tools for metagenomic analysis, covering their strengths, limitations, and key applications. We hope it serves as both a resource and inspiration for new ways to analyze metagenomic data.]

- [[Exploring SNP filtering strategies: the influence of strict vs soft core.](https://doi.org/10.1099/mgen.0.001346)] - [2025.1, _Microbial Genomics_] - [[code andd files](https://github.com/rrwick/Core-SNP-filter)]


- [Comparative evaluation of soil DNA extraction kits for long read metagenomic sequencing.](https://doi.org/10.1099/acmi.0.000868.v3) - [2024.9, _Access Microbiology_]


- [Benchmarking short-, long- and hybrid-read assemblers for metagenome sequencing of complex microbial communities](https://doi.org/10.1099/mic.0.001469)- [2024.6, _Microbiology_]

- [[Solving genomic puzzles: computational methods for metagenomic binning.](https://doi.org/10.1093/bib/bbae372)] - [2024.7, _Briefings in Bioinformatics_]

- [[Introduction to the principles and methods underlying the recovery of metagenome-assembled genomes from metagenomic data.](https://doi.org/10.1002/mbo3.1298)] - [2022.6, _Microbiology Open_]

- [[Impact of microbial genome completeness on metagenomic functional inference.](https://doi.org/10.1038/s43705-023-00221-z)] - [2023.2, _ISME Commun_]

- [A framework for understanding collective microbiome metabolism.]() - [2024.11, _Nat Microbiol_]

### tutorials
- [2025-sourmash-analyzing-genomes-and-metagenomes](https://github.com/ctb/2025-sourmash-analyzing-genomes-and-metagenomes)
- [creating colorful Phylogenetic Trees using a Free web-based tool](https://t.co/2nQo1X0d56)
- [[Heng Li blogs](https://lh3.github.io)]

- [[Ryan Wick blogs](https://rrwick.github.io)]