## implemented workflows and platforms for **metagenomes**
- [[Aviary](https://github.com/rhysnewell/aviary)] - [Python, v0.9.2, 2024.09] - [a snakemake pipeline for assembly, binning, annotation, and strain diversity analysis.]
- [[MAG](https://github.com/nf-core/mag)] - [Nextflow, v3.0.3, 2024.8] - [2022.2, _NAR Genom Bioinform_] - [nf-core/mag: a best-practice pipeline for metagenome hybrid assembly and binning.](https://doi.org/10.1093/nargab/lqac007)
- [[MetaWRAP](https://github.com/bxlab/metaWRAP)] - [Shell/Python, v1.3, 2020.08] - [2018.09, _Microbime_] - [[MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis.](https://doi.org/10.1186/s40168-018-0541-1)]
- [[VEBA](https://github.com/jolespin/veba)] - [Python, v2.3.0, 2024.9] - [2022.10, _BMC Bioinformatics_] - [VEBA: a modular end-to-end suite for in silico recovery, clustering, and analysis of prokaryotic, microeukaryotic, and viral genomes from metagenomes.](https://doi.org/10.1186/s12859-022-04973-8)
- [[ATLAS](https://github.com/metagenome-atlas/atlas)] - [Python, v2.19.0, 2024.7] - [2020.6, _BMC Bioinformatics_] - [ATLAS: a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data.](https://doi.org/10.1186/s12859-020-03585-4)
- [[anvi'o](https://github.com/merenlab/anvio/tree/v8)] - [Win/MacOS/Linux, v8, 2023.09] - [2020.12, _Nat Microbiol_] - [[Community-led, integrated, reproducible multi-omics with anvi’o.](https://doi.org/10.1038/s41564-020-00834-3)]
- [[nano-rave](https://github.com/sanger-pathogens/nano-rave)] - [Nextflow, v1.0.0, 2023.01] - [2023.11, _Nat Microbiol_] - [[Drug resistance and vaccine target surveillance of Plasmodium falciparum using nanopore sequencing in Ghana.](https://doi.org/10.1038/s41564-023-01516-6)]
- [[metagWGS](https://forgemia.inra.fr/genotoul-bioinfo/metagwgs)] - [Nextflow, v2.4.2, 2023.6] - [2024.9, _bioRxiv_] - [metagWGS, a comprehensive workflow to analyze metagenomic data using Illumina or PacBio HiFi reads.](https://doi.org/10.1101/2024.09.13.612854)
- [SqueezeMeta](https://github.com/jtamames/SqueezeMeta) - [C/C++/Python/Perl, v1.6.5, 2024.8] - [2019.1, _Front Microbiol_] - [[SqueezeMeta, A Highly Portable, Fully Automatic Metagenomic Analysis Pipeline](https://doi.org/10.3389/fmicb.2018.03349)]
- [slamM](https://github.com/Ecogenomics/slamM/tree/master) - [Snakemake]
- [[CAT_pack](https://github.com/MGXlab/CAT_pack)] - [Python, v6.0.1, 2024.3] - [2019.10, _Genome Biol_] - [[Robust taxonomic classification of uncharted microbial sequences and bins with CAT and BAT.](https://doi.org/10.1186/s13059-019-1817-x)]


## implemented workflows and platforms for **isolates**
- [[Bactopia](https://github.com/bactopia/bactopia)] - [Nextflow/Perl, v3.1.0, 2024.9] - [2020.8, mSystems] - [[Bactopia: a flexible pipeline for complete analysis of bacterial genomes.](https://doi.org/10.1128/mSystems.00190-20)]
- [ASA³P](https://github.com/oschwengers/asap) - [Groovy/JS, v1.3.0, 2020.5] - [2020.5, _PLoS Comput Biol_] - [[ASA³P: An automatic and scalable pipeline for the assembly, annotation and higher-level analysis of closely related bacterial isolates.](https://doi.org/10.1371/journal.pcbi.1007134)]
- [[microPIPE](https://github.com/BeatsonLab-MicrobialGenomics/micropipe)] - [html tutorial Nextflow, step by step] - [2021.6, _BMC Genomics_] - [[MicroPIPE: validating an end-to-end workflow for high-quality complete bacterial genome construction.](https://doi.org/10.1186/s12864-021-07767-z)]
- [AQUAMIS](https://gitlab.com/bfr_bioinformatics/AQUAMIS) - [Python/Shell, v1.4.2, 2024.6] - [2021.4, _Genes_] - [[Species-Specific Quality Control, Assembly and Contamination Detection in Microbial Isolate Sequences with AQUAMIS.](https://doi.org/10.3390/genes12050644)]
- [Nullarbor](https://github.com/tseemann/nullarbor) - [Perl, v1.41, 2018.6] - [Pipeline to generate complete public health microbiology reports from sequenced isolates.]
- [[ProkEvo](https://github.com/npavlovikj/ProkEvo)] - [jupyter tutorial] - [2021.5, _PeerJ_] - [[ProkEvo: an automated, reproducible, and scalable framework for high-throughput bacterial population genomics analyses.](https://doi.org/10.7717/peerj.11376)]
- [[Public Health Bioinformatics](https://github.com/theiagen/public_health_bioinformatics)] - [WDL, v2.2.1, 2024.9] - [2023.7, _Microbial Genomics_] - [[Accelerating Bioinformatics Implementation in Public Health.](https://doi.org/10.1099/mgen.0.001051)] - [TheiaCoV(viral), TheiaProk(bacterial)]
- [[Public Health Bioinformatics](https://github.com/theiagen/public_health_bioinformatics)] - [WDL, v2.2.1, 2024.9] - [2023.8, _Front. Public Health_] - [[TheiaEuk: A Species-Agnostic Bioinformatics Workflow for Fungal Genomic Characterization.](https://doi.org/10.3389/fpubh.2023.1198213)]
- [[rMAP](https://github.com/GunzIvan28/rMAP)] - [Shell/Python, tutorial] - [2021.6, _Microbial Genomics_] - [[rMAP: the Rapid Microbial Analysis Pipeline for ESKAPE bacterial group whole-genome sequence data.](https://doi.org/10.1099/mgen.0.000583)]
- [[TORMES](https://github.com/nmquijada/tormes)] - [Shell, v1.3.0, 2021.8] - [2019.4, _Bioinformatics_] - [[TORMES: an automated pipeline for whole bacterial genome analysis.](https://doi.org/10.1093/bioinformatics/btz220)]
